docs/maps/maps-getting-started.asciidoc

```diff
@@ -0,0 +1,183 @@
+[[maps-getting-started]]
+== Getting started with Maps
+
+*Maps* is your place for analyzing geospatial data. For example, a map might 
```

Comment:
Not sure an example is needed in this paragraph. The paragraph specifies a similar example. Maybe just get ride of this first paragraph? The maps app is already defined in the top level docs page